tests/utils.go

```diff
@@ -3135,6 +3135,10 @@ func NewRepeatableVirtctlCommand(args ...string) func() error {
 func ExecuteCommandOnPod(virtCli kubecli.KubevirtClient, pod *k8sv1.Pod, containerName string, command []string) (string, error) {
 	stdout, stderr, err := ExecuteCommandOnPodV2(virtCli, pod, containerName, command)
 
+	if err != nil && len(stderr) > 0 {
+		return "", fmt.Errorf("err: %v, stderr: %v", err, stderr)
```

Comment:
Normally at go we nest the errors with `:` so they are like "error 1: error 2: error 3", let's trait stderr like another error and add "error" at the end.
```golang
fmt.Errorf("failed executing command  on pod: %v: %v", stderr, err)
```

Reply:
And keep the other failures in this method as is?

Reply:
Think it about it again, I think we have to print stdout too is good for debugging and in this case is not a bad thing to state taht we are pringing "stderr" so what I mean is the following 

```golang
if err != nil {
   err = fmt.Errorf("failed executing command on pod: %v", err)
   if len(stderr) > 0 {
		err = fmt.Errorf("%v: %s stderr: %s: stdout: %s", err, stdout, stderr)
   }
   return err
}
```

Reply:
done. In case of an error I'm printing `stderr` and `stdout` even if they are empty.